backend/src/main/java/eventlistener/service/notificationuser/NotificationUserMapper.java

```diff
@@ -13,14 +12,14 @@
 @Service
 public class NotificationUserMapper {
 
-    public NotificationUser mapDTO(NotificationUserDTO dtoToMap){
+    public NotificationUser mapNotificationUser(NotificationUserDTO dtoToMap){
```

Comment:
```suggestion
    public static NotificationUser mapNotificationUser(NotificationUserDTO dtoToMap){
```
and all of the other mapper methods too, as long as you don't use any non static methods in them like this you don't have to instatiate the mappers but can access their methods directly